3.analysis/scripts/nbconverted/3.t_tests.py

```diff
@@ -207,3 +208,13 @@
 # sort to show most significant p-values
 t_test_results_df.sort_values(by='p_value', ascending = True)
 
+
+# In[12]:
+
+
+# ANOVA Testing
+f_statistic, p_value = f_oneway(adult_male_latent_df_float, ped_male_latent_df_float, adult_female_latent_df_float, ped_female_latent_df_float)
```

Comment:
Consider also adding a post-hoc test for the anova test. This will tell you which groups are different from eachother. Post-hoc would be run if the anova test comes back significant.